src/builder/Actions.sol

```diff
@@ -18,6 +18,8 @@ library Actions {
     string constant ACTION_TYPE_BRIDGE = "BRIDGE";
     string constant ACTION_TYPE_TRANSFER = "TRANSFER";
 
+    uint256 constant TRANSFER_EXPIRY_BUFFER = 10_000;
```

Comment:
10k is totally arbitrary

Reply:
Hans suggested 7 days; we could update this to `7 * 24 * 60 * 60` or `7 days`

Reply:
Sure, 7 days makes sense.